public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -222,11 +221,11 @@ export function FolderAndGroup({ initialFolder }: FolderAndGroupProps) {
               <AsyncSelect
                 disabled={!folder}
                 inputId="group"
-                key={`my_unique_select_key__${folder?.title ?? ''}`}
+                key={`my_unique_select_key__${selectedGroup?.title ?? ''}`}
                 {...field}
                 loadOptions={debouncedSearch}
-                loadingMessage={t('folder-picker.loading', 'Loading folders...')}
-                defaultOptions
+                loadingMessage={'Loading groups...'}
```

Comment:
Why was `t('folder-picker.loading', 'Loading folders...')` removed? I see `t` is used in a few other places. I don't know if we should apply it for user visible texts everywhere or not.

Reply:
I removed this in this commit because I added it wrongly. I think we agreed with @gillesdemey not to add new ones until we decide applying the translation everywhere.